proposals/20200913-rootlessKubeflow.md

```diff
@@ -0,0 +1,35 @@
+# Rootless Kubeflow
+
+### Goals
+
+We want to run Kubeflow 99 % rootless accoring to CNCF/Kubernetes best practices.
+
+The main steps are adding an additional profile for istio-cni or ambient mesh, updating the documentation and manifest generation process. 
+Then adding the baseline and restricted PSS as kustomize component to /contrib and extending the profile controller to annotate user namespaces with configurable PSS labels.
+
+We want to use a staged approach.
+First Stage:
+
+1. Implement Istio 1.17.5 and use it by default. This is important for the Kubeflow 1.8  feature freeze
```

Comment:
Is it more appropriate to say istio version > 1.17? and can you specify why > 1.17